index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous" />
+
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+  <header id="header">
+    <!--<img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" height="180" alt=""-->
+    <nav id="nav-bar">
+      <div class="topnav">
+        <a class="nav-link" href="#header">
+          <div class="logo-image">
+            <img id="header-img"
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="img"
+              class="img-fluid">
+          </div>
+        </a>
+        <a class="nav-link" href="#header">Home</a>
+        <a class="nav-link" href="#header">News</a>
+        <a class="nav-link" href="#header">Contact</a>
+      </div>
+    </nav>
+  </header>
+
+  <div class="pricing-header px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+
+    <video id="video" controls="controls" src="media/shuttle.mp4">
+      Your browser does not support the HTML5 Video element.
+    </video>
+
+    <form action="https://www.freecodecamp.com/email-submit" method="POST" id="form">
+      <input type="email" id="email" name="email" placeholder="Email">
+      <input type="submit" id="submit" value="submit">
+    </form>
+
+  </div>
+
+
+
+
+  <!--
+
+    Hello Camper!

```

Comment:
- [ ] you can remove this part of the code. Because those lines are unnecessary and you can get confused :blush: